### PR TITLE
feat: [DHIS2-14125] improve no saved list 

### PR DESCRIPTION
**Pull request by jasminenguyennn**
Screenshot 2022-12-07 at 13 24 19

### CODE REVIEW

File: src/core_modules/capture-core/components/Pages/Search/TemplateSelector/TemplateSelector.component.js [hunk 1]
```diff
@@ -63,15 +85,29 @@ const TemplateSelectorPlain = (props: Props) => {
...
+            <Widget
+                header={i18n.t('Saved lists in this program')}
+                open={open}
+                onClose={() => setOpen(false)}
+                onOpen={() => setOpen(true)}
+            >
+                <div className={classes.innerContainer}>
+                    <BookmarkAddIcon className={classes.icon} />
+                    <p className={classes.text}>
+                        {i18n.t('Saved list offer quick access to your most used views in a program.')}
```
Comment (cooper-joe) [inline src/core_modules/capture-core/components/Pages/Search/TemplateSelector/TemplateSelector.component.js:97]:
```suggestion
                        {i18n.t('Saved lists offer quick access to your most used views in a program.')}
```

File: src/core_modules/capture-core/components/Pages/Search/TemplateSelector/TemplateSelector.component.js [hunk 2]
```diff
@@ -63,15 +85,29 @@ const TemplateSelectorPlain = (props: Props) => {
...
+                    </p>
+                    {customTemplates.length > 0 ? (
+                        <div className={classes.configsContainer}>{configElements}</div>
+                    ) : (
+                        <p className={classes.text}>
+                            {i18n.t('There are no saved list in this program yet, create one using the button below.')}
```
Comment (cooper-joe) [inline src/core_modules/capture-core/components/Pages/Search/TemplateSelector/TemplateSelector.component.js:103]:
```suggestion
                            {i18n.t('There are no saved lists in this program yet, create one using the button below.')}
```

File: src/core_modules/capture-core/components/Pages/Search/TemplateSelector/TemplateSelector.component.js (hunk 1 above)
Comment (cooper-joe) [inline src/core_modules/capture-core/components/Pages/Search/TemplateSelector/TemplateSelector.component.js:97]:
```suggestion
                        {i18n.t('Saved lists offer quick access to your most used views in a program.')}
```

File: i18n/en.pot [hunk 3]
```diff
@@ -906,8 +906,18 @@ msgstr "Create new"
 msgid "Saved lists in this program"
 msgstr "Saved lists in this program"
 
-msgid "Create custom working list"
-msgstr "Create custom working list"
+msgid "Saved list offer quick access to your most used views in a program."
+msgstr "Saved list offer quick access to your most used views in a program."
```
Comment (cooper-joe) [inline i18n/en.pot:910]:
```suggestion
msgid "Saved lists offer quick access to your most used views in a program."
msgstr "Saved lists offer quick access to your most used views in a program."
```

File: i18n/en.pot [hunk 4]
```diff
@@ -906,8 +906,18 @@ msgstr "Create new"
...
+
+msgid ""
+"There are no saved list in this program yet, create one using the button "
+"below."
+msgstr ""
+"There are no saved list in this program yet, create one using the button "
+"below."
```
Comment (cooper-joe) [inline i18n/en.pot:917]:
```suggestion
"There are no saved lists in this program yet, create one using the button "
"below."
msgstr ""
"There are no saved lists in this program yet, create one using the button "
"below."
```

**Review by cooper-joe — CHANGES_REQUESTED** [review]

**Review by cooper-joe — APPROVED** [review]

**Review by eirikhaugstulen — COMMENTED** [review]
Hey, @jasminenguyennn!

I notice that the ui is a bit different than what is shown in the design guide. It looks like the picture expects there to be two widgets.
Has the issue changed or is this cleared?

**Comment by jasminenguyennn** [comment]
> Hey, @jasminenguyennn!
> 
> I notice that the ui is a bit different than what is shown in the design guide. It looks like the picture expects there to be two widgets. Has the issue changed or is this cleared?

good catch @eirikhaugstulen, I update the branch with the suggestion.

**Review by eirikhaugstulen — APPROVED** [review]
Looks good @jasminenguyennn 👍

**Review by geethaalwan — APPROVED** [review]
Tested successfully on 2.40 version

**Pull request merged**